nxrm-onprem-ha-helm/Chart.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v2
+name: nxrm-ha
+description: Highly available deployment of Sonatype Nexus Repository
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 58.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: 3.58.1
+
+keywords:
+  - artifacts
+  - dependency
+  - management
+  - sonatype
+  - nexus
+  - repository
+  - quickstart
+  - ci
+  - repository-manager
+  - nexus3
+home: https://www.sonatype.com/nexus-repository-oss
```

Comment:
Given this is HA, we should link to something pro perhaps:

https://www.sonatype.com/products/sonatype-nexus-repository

Reply:
https://github.com/sonatype/nxrm3-ha-repository/pull/17/commits/ee3f058756e4c598e1da1dae6682d4a0bdd26bf0